### PR TITLE
Fix panic after getting terminal width in some CI environments

### LINKED ISSUE #1579: K6 crashes with "panic: runtime error: integer divide by zero" when run in CodeFresh
## Environment
<!--- Provide as much information about your environment as possible: output from `k6 version`,
      OCI runtime and image information if using containers, cloud execution environment, etc. -->
- k6 version: 0.27.0 / 0.27.1
- OS and version: CodeFresh
- Docker version and image, if applicable: loadimpact/k6:0.27.0 and loadimpact/k6:0.27.1


## Expected Behavior
<!--- Tell us what should happen -->
k6 should successfully run load tests in a container

## Actual Behavior
<!--- Tell us what happens instead of the expected behavior -->
k6 crashes with the following stack trace:
```
execution: local                                                                                                                                                                                                               
     script: /scripts/debug.bundle.js                                                                                                                                                                                            
     output: json=test_results.json                                                                                                                                                                                              
                                                                                                                                                                                                                                 
  scenarios: (100.00%) 1 executors, 10 max VUs, 1m30s max duration (incl. graceful stop):                                                                                                                                        
           * default: Up to 10 looping VUs for 1m0s over 3 stages (gracefulRampDown: 30s, gracefulStop: 30s)                                                                                                                     
                                                                                                                                                                                                                                 
panic: runtime error: integer divide by zero--] Init VUs                                                                                                                                                                         
                                                                                                                                                                                                                                 
goroutine 6 [running]:                                                                                                                                                                                                           
github.com/loadimpact/k6/cmd.renderMultipleBars(0xc007600101, 0x7, 0x0, 0x0, 0xc002678000, 0x2, 0x2, 0x5327a4, 0xd1fce0, 0x0)                                                                                                    
        github.com/loadimpact/k6/cmd/ui.go:218 +0xda9                                                                                                                                                                            
github.com/loadimpact/k6/cmd.showProgress.func2(0xc003713101)                                                                                                                                                                    
        github.com/loadimpact/k6/cmd/ui.go:283 +0x8c                                                                                                                                                                             
github.com/loadimpact/k6/cmd.showProgress(0x10d8a40, 0xc000d94100, 0x0, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc00159cd80, ...)                                                                                                         
        github.com/loadimpact/k6/cmd/ui.go:352 +0x52a                                                                                                                                                                            
github.com/loadimpact/k6/cmd.glob..func11.1(0x10d8a40, 0xc000d94100, 0xc0005d6300, 0xc0009a2500, 0xc0001b2000, 0xc000ec4bd0)                                                                                                     
        github.com/loadimpact/k6/cmd/run.go:190 +0xb9                                                                                                                                                                            
created by github.com/loadimpact/k6/cmd.glob..func11                                                                                                                                                                             
        github.com/loadimpact/k6/cmd/run.go:189 [+0xd34]
```

## Steps to Reproduce the Problem
This seems similar to https://github.com/loadimpact/k6/issues/1559 as it is unable to get the terminal width correctly, but 0.27.1 did not fix the issue in CodeFresh, and still crashes in the same way.  Reverting to 0.26.2 works as expected.

Attempting to run any script on a k6 0.27.0 or 0.27.1 container in CodeFresh should see this behavior.

### PR DESCRIPTION
**Pull request by imiric**
In some CI environments like Codefresh, k6 detects that it's running in a TTY, the err returned by `terminal.GetSize()` is `nil`, yet `termWidth` is 0. Strange scenario, and we should fix our TTY check if possible, but it shouldn't panic with `divide by zero` anymore.

Closes #1579

### CODE REVIEW

**Comment by imiric** [comment]
After this fix, runs on Codefresh show:

```
WARN[0001] error getting terminal size                   error="<nil>"
```

And the jobs pass successfully, though there's another issue actually making requests from Codefresh:

```
WARN[0001] Request Failed                                error="Get \"https://test-api.k6.io/\": x509: certificate signed by unknown authority"
```

But that can presumably be resolved by updating ca-certificates, though I didn't troubleshoot it further.

File: cmd/ui.go [hunk 1]
```diff
@@ -333,19 +337,19 @@ func showProgress(
...
 			return
 		case <-winch:
-			if !errTermGetSize {
+			if stdoutTTY && !errTermGetSize {
 				// More responsive progress bar resizing on platforms with SIGWINCH (*nix)
-				termWidth, _, err = terminal.GetSize(fd)
-				if err != nil {
-					termWidth = defaultTermWidth
+				tw, _, err := terminal.GetSize(fd)
+				if tw > 0 && err == nil {
```
Comment (imiric) [inline cmd/ui.go:343]: This check might be overkill, but better safe than sorry in this case.

**Comment by luketn** [comment]
Hi, I have the same error reported in #1579 and addressed here running a docker-compose CI build of K6:
```
Status: Downloaded newer image for loadimpact/k6:latest

          /\      |‾‾|  /‾‾/  /‾/   
     /\  /  \     |  |_/  /  / /    
    /  \/    \    |      |  /  ‾‾\  
   /          \   |  |‾\  \ | (_) | 
  / __________ \  |__|  \__\ \___/ .io

 Init   [--------------------------------------] runner
  execution: ---------------------------------] Init API serverKlocal
     script: scripts/ewoks.js
     output: influxdb=http://influxdb:8086/k6 (http://influxdb:8086)

  scenarios: (100.00%) 1 executors, 5 max VUs, 50s max duration (incl. graceful stop):
           * default: Up to 5 looping VUs for 20s over 3 stages (gracefulRampDown: 30s, gracefulStop: 30s)

panic: runtime error: integer divide by zero--] Init VUs

goroutine 10 [running]:
github.com/loadimpact/k6/cmd.renderMultipleBars(0xc001680101, 0x7, 0x0, 0x0, 0xc001c02890, 0x2, 0x2, 0x5327a4, 0xd1fce0, 0x0)
	github.com/loadimpact/k6/cmd/ui.go:218 +0xda9
github.com/loadimpact/k6/cmd.showProgress.func2(0xc0004a5101)
	github.com/loadimpact/k6/cmd/ui.go:283 +0x8c
github.com/loadimpact/k6/cmd.showProgress(0x10d8a40, 0xc001df5440, 0x0, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc001f06600, ...)
	github.com/loadimpact/k6/cmd/ui.go:352 +0x52a
github.com/loadimpact/k6/cmd.glob..func11.1(0x10d8a40, 0xc001df5440, 0xc002077500, 0xc002556a00, 0xc000092000, 0xc002d13180)
	github.com/loadimpact/k6/cmd/run.go:190 +0xb9
created by github.com/loadimpact/k6/cmd.glob..func11
	github.com/loadimpact/k6/cmd/run.go:189 +0xd34
```

Could you also let us know when the Docker Hub image will be released for 0.28 with this fix in it? Until then I am having to use the bleeding edge \:master tag, which I would prefer not to do.

I encountered the bug with the circleci config:
```
version: 2
jobs:
  build:
    machine: true
    steps:
      - checkout
      - run: |
          docker-compose up -d influxdb grafana
          docker-compose run k6 run script.js
```

(so this was not limited to CodeFresh CI) 

I can confirm that running the same script with the K6 image 
`loadimpact/k6:master`

Fixes the issue.

**Comment by imiric** [comment]
Hi @luketn, sorry about that. We had some issues with the v0.28.0 release that caused the Docker image, Linux and Windows packages to not be built and published.

This is now addressed, and you can pull the `loadimpact/k6:0.28.0` image now. Cheers!

**Pull request merged**